Automattic/wp-calypso
# Help: course query

This pull request adds a new data component that queries the wpcom /help/courses api end point. In doing so it:
- Removes the function that returned hard coded course information.
- Altered the state tree by adding `requesting` at `state.help.courses`.
### How to test
1. Login as a business plan user.
2. Navigate to https://calypso.live/help/courses?branch=add/help-courses-query
3. Notice that the list of courses is still loading as expected

![screen shot 2016-10-03 at 8 06 08 pm](https://cloud.githubusercontent.com/assets/1854440/19058389/dfda4b20-89a4-11e6-8985-8f678ce1f418.png)
